processing/src/main/java/org/apache/druid/java/util/metrics/cgroups/CpuSet.java

```diff
@@ -72,7 +72,7 @@ private int[] readCpuSetFile(String file)
       output = lines.stream().map(this::parseStringRangeToIntArray).findFirst().orElse(output);
     }
     catch (RuntimeException | IOException ex) {
-      LOG.error(ex, "Unable to read %s", file);
+      LOG.noStackTrace().warn(ex, "Unable to read %s, these metrics will be skipped", file);
```

Comment:
Nit: IMHO, it might be better to add all the `CPUS_FILE`, `EFFECTIVE_CPUS_FILE` etc in an array and then replace `these` with the exact metric name. That would help admins to figure out what happens with `CPUS_FILE` since the cluster admin would most likely grep for the particular string.

Reply:
@cryptoe , The file name will be mentioned here in the log already, just in separate log lines.